translations/test/en_US/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,30 @@
+# Test translations for unit tests.
+# Copyright (C) 2016
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR brendan.d.burns@gmail.com, 2016.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gettext-go-examples-hello\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2013-12-12 20:03+0000\n"
+"PO-Revision-Date: 2016-12-13 22:12-0800\n"
+"Last-Translator: Brendan Burns <brendan.d.burns@gmail.com>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.6.10\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Language-Team: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: en\n"
+
+msgctxt "test_string"
+msgid "test_string"
+msgstr "baz"
```

Comment:
Yeah, I want this for the unit test, so I can try explicit language loading instead of the default.

The reason it is different so that the unit test won't pass if we are accidentally loading the "default" language for some reason.

I want `translate("default", "test_string") != translate("en_US, "test_string")` otherwise a unit test might accidentally 'pass' even though the code is actually wrong.